packages/compass-schema/src/modules/index.js

```diff
@@ -3,13 +3,11 @@ import boolean from './boolean';
 import date from './date';
 import string from './string';
 import objectid from './date';
-import coordinates from './coordinates';
 
 export default {
   number,
   boolean,
   date,
   string,
   objectid,
-  coordinates,
```

Comment:
Is this part of this patch? Why it was removed?

Reply:
i added a description to the PR, it pulls in mapbox, which doesn't have a permissive license

Reply:
Ah, missed it, thanks!